pyfarm/models/job.py

```diff
@@ -547,6 +547,36 @@ def alter_frame_range(self, start, end, by):
             if self.state != WorkState.RUNNING:
                 self.state = None
 
+    def rerun(self):
```

Comment:
Small doc string would be nice so you can tell what the method does on readthedocs.